install-tensorflow_cc.md

```diff
@@ -1,12 +1,12 @@
 # Install tensorflow_cc
 
-We use `tensorflow_cc` for Tensorflow C++ API without using Bazel.Use `cmake` to build your project using `tensorflow_cc` as library. Official page: [link](https://github.com/FloopCZ/tensorflow_cc)
+We use `tensorflow_cc` for Tensorflow C++ API. Use `cmake` to build your project using `tensorflow_cc` as library. Official page: [link](https://github.com/FloopCZ/tensorflow_cc)
 
-- [Install tensorflow_cc (Ubuntu 18.04)](#install-tensorflow_cc-(Ubuntu-18.04))
-- [Install tensorflow_cc (Ubuntu 14.04 and 16.04)](#install-tensorflow_cc-(Ubuntu-14.04-and-16.04))
+- [Install tensorflow_cc (Ubuntu 18.04)](#ubuntu-18.04)
```

Comment:
You can easily get the section anchor by clicking on the anchor button that appears when you hover the cursor over the section title. Here, use `#install-tensorflow_cc-ubuntu-1804` and delete the dummy `<a>` element further down. Same thing applies to the 14.04&16.04 section.